Ranking Criteria - Add Performances to Allowances (#7261)
ppy/osu-wiki: wiki/Ranking_Criteria/en.md

- **For songs where the composer(s) and singer(s) are different people, the singer(s) may be listed after the composer(s) or circle/group name following a `feat.` indicator.**
- **If a Unicode song title or artist has an official translation or romanisation provided by the artist, it may be used in the respective romanised field. If both a translation and romanisation are available, either may be used.**
- **If a beatmap's track was contributed to by multiple artists, they may be listed with commas in between.** If there are 3 or more contributing artists and they are not part of one officially labelled group, `Various Artists` or other descriptive artist labels may be used instead.

> - **For remixes, covers, or performances, the original artist may be used in the artist field, as long as the title field is modified to clearly show that the song is remixed.** This marker should all be in parentheses and contain the remix/cover artist followed by descriptor.

Reviewer: ```suggestion
- **For remixes, covers, or performances, the original artist may be used in the artist field, as long as the title field is modified to clearly show that the song is become different.** This marker should all be in parentheses and contain the remix/cover artist followed by descriptor.
```

Reviewer: `remixes` is not stands on this anymore. perhaps could be spread to better wording.

> - **For remixes, covers, or performances, the original artist may be used in the artist field, as long as the title field is modified to clearly show that the song is become different.** This marker should all be in parentheses and contain the remix/cover/performances artist followed by descriptor.

Reviewer 2: let's also clean up a few grammatical errors

```suggestion
- **For remixes, covers, or performances, the original artist may be used in the artist field, as long as the title field is modified to clearly show that the song is not the original version.** This marker should be in parentheses and contain the remix/cover artist or the performer, followed by a descriptor.
```

not entirely sure what "descriptor" is supposed to mean here

Author: a label which describes what the song is, such as Cover, Remix, Rearrange, Jazz Remix, etc. Usually based off the song or how it may be labelled besides its direct title.

> - **For remixes, covers, or performances, the original artist may be used in the artist field, as long as the title field is modified to clearly show that the song is not the original version.** This marker should be in parentheses and contain the remix/cover artist or the performer, followed by a descriptor.

Reviewer 2: > a label which describes what the song is, such as Cover, Remix, Rearrange, Jazz Remix, etc. Usually based off the song or how it may be labelled besides its direct title.

ah. from the thread i see `(Perf. {artist})` being suggested, which would technically be discouraged by this wording since it states the descriptor should follow the performer. i'm not sure how standardised these types of markers are but i thought to point it out

```suggestion
- **For remixes, covers, or performances, the original artist may be used in the artist field, as long as the title field is modified to clearly show that the song is not the original version.** This marker should be in parentheses and contain the remix/cover artist or the performer as well as a descriptor.
```
